data/dataset-curation.md

```diff
@@ -50,8 +50,20 @@ sub-001
 
 Many kinds of data have a place specified for them by BIDS. See [file naming conventions](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure) and the [MRI](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) and [Microscopy](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/10-microscopy.html) extensions for full details.
 
-```{warning}
-TODO: describe neuropoly-specific BIDS entities, like bp-cspine or acq-MTon
+```{note}
+If you need to differentiate spinal cord images from the brain, use the `acq-cspine` tag. For example, `sub-001_acq-cspine_T1w.nii.gz`.
```

Comment:
Adding a link to the BIDS website with example of template would be useful (to verify our document is always up to date and avoid https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/94/files#r1085514220)

Reply:
Addressed in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/94/commits/6e1ea74d61142bd9af902e09256269aba8d45286